mne/channels/layout.py

```diff
@@ -907,19 +907,23 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         raise ValueError("'unit' should be either 'm', 'cm', or 'mm'.")
     if transform is True:
```

Comment:
Also a point not blamed on @haribharadwaj: Python users might be used to being able to use `transform=1` so it might be more user friendly to say `if transform` instead of `if transform is True`